src/aggregates/vector_selector.rs

```diff
@@ -0,0 +1,493 @@
+use pgx::*;
+
+use pgx::error;
+
+use crate::aggregate_utils::in_aggregate_context;
+use crate::aggregates::{Milliseconds, STALE_NAN, USECS_PER_MS};
+use serde::{Deserialize, Serialize};
+
+use crate::palloc::{Inner, InternalAsValue, ToInternal};
+use crate::raw::{bytea, TimestampTz};
+
+// a vector selector aggregate has the same semantics as parse.VectorSelector processing
```

Comment:
We should probably change comments like these to be doc comments, i.e. `///` rather than `//`, and then control whether or not they show up in the generated crate docs using doc attributes, e.g. `#[doc(hidden)]`

Reply:
Agree. I have done this already in some places. IMO it requires more than just converting a `//` to a `///`, as I think that the quality of the code documentation needs to be improved. I've opened #42 for this.